db/v-2/tables/lnk-project-github.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="create-lnk-project-github" author="sanyavertolet" context="dev or prod">
+        <createTable tableName="lnk_project_github">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="project_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="owner" type="varchar(250)">
+                <constraints nullable="false"/>
```

Comment:
what's about a foreign key to user?

Reply:
Why do we need it? Later `ProjectPermissionEvaluator` will be used